gulpfile.js

```diff
@@ -62,7 +62,10 @@ gulp.task('revreplace', function() {
         const src = img.attr('data-src') || img.attr('data-org');
         if (!src) return;
 
-        const jpgPath = replaceBackSlash(rename(src, {extname: '.jpeg'}));
+        // url encode the image path to handle cases where there is a space in image name
+        const srcEncoded = src.replace(' ', '%20');
```

Comment:
Not sure, but maybe [encodeURI](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI) is better than this code.

Reply:
Will test asap

Reply:
It worked. Thank you